templates/ansible/README.md.erb

```diff
@@ -0,0 +1,18 @@
+<% autogen_exception -%>
+# Google Cloud Platform Ansible Collection
+This collection provides a series of Ansible modules and plugins for interacting with the [Google Cloud Platform](https://cloud.google.com)
+
+# Installation
+```bash
+ansible-galaxy collection install google.cloud
```

Comment:
Since this won't work until 2.9, how about adding some generic comment like "For Ansible 2.9+" or something?

Have you verified that this is available in the 2.9 beta release?

Reply:
This is available in 2.9.